src/main/java/racingcar/controller/RacingGameWebController.java

```diff
@@ -3,28 +3,36 @@
 import java.util.List;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import racingcar.domain.Cars;
 import racingcar.domain.TryCount;
+import racingcar.dto.GameHistoriesResponseDto;
 import racingcar.dto.GameResultResponseDto;
 import racingcar.dto.StartGameRequestDto;
 import racingcar.service.RacingCarService;
-import racingcar.utils.CarsFactory;
 
+@RequestMapping("/plays")
 @RestController
-public class RacingGameController {
+public class RacingGameWebController {
 
-    private static final String CAR_NAME_SEPARATOR = ",";
+    private static final String CAR_NAME_DELIMITER = ",";
 
     private final RacingCarService racingCarService;
 
-    public RacingGameController(final RacingCarService racingCarService) {
+    public RacingGameWebController(final RacingCarService racingCarService) {
         this.racingCarService = racingCarService;
     }
 
-    @PostMapping("/plays")
+    @GetMapping
+    public ResponseEntity<List<GameHistoriesResponseDto>> findGameHistories() {
+        return new ResponseEntity<>(racingCarService.findAllGameHistories(), HttpStatus.OK);
```

Comment:
ResponseEntity의 경우 `new`로 생성하는 경우보다 Builder 클래스를 활용하는 경우가 더 많으니 알아두시면 좋을것같아요.

```suggestion
        return ResponseEntity.ok(racingCarService.startRace(cars, tryCount));
```

Reply:
확실히 생성자로 했을 때보다, 빌더 패턴으로 변경하니 가독성이 좋아졌음을 느낍니다!

또한, `ResponseEntity`의 매개변수 순서도 외울 필요가 없을 것 같습니다 :)

감사합니다.